servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/RoundRobinLoadBalancer.java

```diff
@@ -169,7 +169,7 @@ public void onNext(final ServiceDiscovererEvent<ResolvedAddress> event) {
                 final List<Host<ResolvedAddress, C>> usedAddresses =
                     usedHostsUpdater.updateAndGet(RoundRobinLoadBalancer.this, oldHosts -> {
                         if (oldHosts == CLOSED_LIST) {
-                            return CLOSED_LIST;
+                            return oldHosts;
```

Comment:
Intellij IDEA had a warning that there is no cast if `CLOSED_LIST` is used, easier just to return `oldHosts` and not worry about types.